<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateProject</class>
 <widget class="QDialog" name="CreateProject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="le_proName">
   <property name="geometry">
    <rect>
     <x>133</x>
     <y>73</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_proDir">
   <property name="geometry">
    <rect>
     <x>133</x>
     <y>42</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_record">
   <property name="geometry">
    <rect>
     <x>133</x>
     <y>104</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="le_frameRate">
   <property name="geometry">
    <rect>
     <x>133</x>
     <y>135</y>
     <width>61</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="chb_up">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>43</y>
     <width>63</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Upper</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chb_rgb">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>74</y>
     <width>52</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>RGB</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chb_depth">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>105</y>
     <width>62</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Depth</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="chb_skelet">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>136</y>
     <width>64</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Skelet</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>42</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Project Directory</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>73</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Project Name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>104</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Record Number</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>135</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Frame Rate</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_selectDir">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>40</y>
     <width>31</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>180</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_create">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>180</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Create</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pb_selectDir</tabstop>
  <tabstop>le_proDir</tabstop>
  <tabstop>le_proName</tabstop>
  <tabstop>le_record</tabstop>
  <tabstop>le_frameRate</tabstop>
  <tabstop>chb_up</tabstop>
  <tabstop>chb_rgb</tabstop>
  <tabstop>chb_depth</tabstop>
  <tabstop>chb_skelet</tabstop>
  <tabstop>pb_create</tabstop>
  <tabstop>pushButton_2</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>CreateProject</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>383</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>454</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
